src/ffi/json.rs

```diff
@@ -35,11 +39,12 @@ pub enum JSON {
         object: HashMap<String, JSON>,
     },
 }
-#[uniffi::export]
+// TODO: This passes cargo build but does not work with uniffi-bindgen.
+//#[uniffi::export]
 impl JSON {
     /// Deserializes JSON serialized to a string to `JSON` or errors with a
     /// [JSONDeserializationError] if the string is invalid JSON.
-    #[uniffi::constructor]
+    //#[uniffi::constructor]
```

Comment:
This makes sense because our understanding of how `uniffi` wants to be used is only `uniffi::Object`s are supposed to have `uniffi::constructors`, so it was a mistake for me to leave this marked as a constructor when `JSON` is a `uniffi::Enum`.  You can delete this line and the TODO. It was a me not updating this correctly after I figured how how `uniffi::Object`s are supposed to work.